src/System.Management.Automation/FormatAndOutput/common/ComplexWriter.cs

```diff
@@ -49,24 +49,14 @@ internal void WriteObject(List<FormatValue> formatValueList)
             // we always start with no indentation
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
XmlCommand DRT basic functionality Tests.Import-Clixml StopProcessing should succeed
```powershell
Expected 'Stopped', but got Completed.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\XMLCommand.Tests.ps1: line 77
77: 		$ps.InvocationStateInfo.State | Should -Be "Stopped"
```